tests/test_bake_project.py

```diff
@@ -165,28 +158,25 @@ def test_quality_with_models(cookies):
 
 def check_quality(result):
     """Run quality tests on the given generated output."""
-    for dirpath, _dirnames, filenames in os.walk(str(result.project)):
-        pylintrc = str(result.project.join('pylintrc'))
+    for dirpath, _dirnames, filenames in os.walk("."):
```

Comment:
Why are we running these commands on individual Python files? That's really slow.

Reply:
Mainly because that's how the cookiecutter we forked did it: https://github.com/pydanny/cookiecutter-djangopackage/blob/master/tests/test_bake_project.py#L238-L248 .  The main advantage is that it works on all generated Python files without needing to update the test configuration if new files or directories are added, but there's probably a better way to do that.